lib/components/pages/home_page.dart

```diff
@@ -67,8 +67,10 @@ class _HomePageState extends State<HomePage> {
     initializeLocalNotifications(_onDidReceiveLocalNotification,
         onSelectNotification: (String payload) {
       if (payload.startsWith("doBiometrics")) {
-        locator<NavigationService>()
-            .navigateTo('alertpagebiometrics', payload.split("-")[1]);
+        locator<NavigationService>().navigateTo(
+            'alertpagebiometrics',
+            LocalNotificationIds.values
```

Comment:
Same way you have an enum I think you can provide a method that abstracts all this ugliness (?), something similar to `LocalNotificationIds.getId(payload)`